article/sections/_2_graph_importance.tex

```diff
@@ -133,79 +121,21 @@ \section{Perils of disregard}
    \label{fig:SR3_causal_2}
 \end{figure}
 
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{WTW_interacting_graph}
-   \caption{Causal Graph for the Walk-Transit-Walk Utility Function}
-   \label{fig:WTW_causal_2}
-\end{figure}
-
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{WTD_interacting_graph}
-   \caption{Causal Graph for the Walk-Transit-Drive Utility Function}
-   \label{fig:WTD_causal_2}
-\end{figure}
-
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{DTW_interacting_graph}
-   \caption{Causal Graph for the Drive-Transit-Walk Utility Function}
-   \label{fig:DTW_causal_2}
-\end{figure}
-
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{WALK_interacting_graph}
-   \caption{Causal Graph for the Shared Ride 3+ Utility Function}
-   \label{fig:WALK_causal_2}
-\end{figure}
-
-\begin{figure}
-   \centering
-   \includegraphics[width=0.2\textwidth]{BIKE_interacting_graph}
-   \caption{Causal Graph for the Bike Utility Function}
-   \label{fig:BIKE_causal_2}
-\end{figure}
+Using the same outcome model, we:
+\begin{itemize}
+   \item Simulate data from the DAGs shown in Figure \ref{fig:IND_GRAPH} through Figure \ref{fig:SR3_causal_2}
+   \item Modify the travel distance variable in all graphs to emulate a company's decision to move its employees closer to campus, and modify all children nodes of the impacted variable (where appropriate) based on the structure of each DAG.
+   \item Predict the probabilities of choosing car-centric modes before and after modifying the data to emulate the proposed intervention and compute the differences in mode choice probabilities based on the different DAGs
+\end{item}
```

Comment:
```suggestion
To show this, we run the following simulation, using the same outcome model:
\begin{itemize}
   \item We simulate data from the DAGs shown in Figure \ref{fig:IND_GRAPH} through Figure \ref{fig:SR3_causal_2}, representing the true data generating model.
   \item In one scenario, we only modify the travel distance variable in all graphs to emulate a company's decision to move its employees closer to campus. This is similar to assuming the DAG in Figure \ref{fig:IND_GRAPH} as the true data generating model.
   \item In the other scenario, we modify the travel distance variable in all graphs, as well as all the variables that are dependent on it based on the DAGs in Figure \ref{fig:IND_GRAPH} through Figure \ref{fig:SR3_causal_2}. This case is meant to reflect the "correct" approach needed to quantify the effects of the companies policy. 
   \item We predict the probabilities of choosing car-centric modes before and after modifying the data to emulate the proposed intervention under each of the two scenarios above, and compute the differences in mode choice probabilities.
\end{item}
```

Reply:
@hassan-obeid, what do you mean by "To show this"? Looking at the lines before 124, there is no clear "this" that I identify.
Lines 96 - 122 are just the images of causal graphs.
Lines 94 and earlier are describing the data.
It seems line 124 is the first line where we begin describing the simulation.
I don't think it's intuitive / appropriate to refer back to something (e.g. "this") about the simulation yet.

Also, can we keep the instruction-style description of our simulation?
I.e., by beginning each bullet with a verb that immediately tells readers what we did. 
In @bouzaghrane's original version, this was done with "Simulate; Modify; Predict".
I think it'd be easy enough to do the same thing here.

Lastly, can we keep each bullet point to only a single idea / action?
This basically means no ", and" in the sentence.
I would split the second bullet point at ", and modify."
I'd split the third bullet point at "and compute".

Reply:
@timothyb0912 the "This" refers to line 93, ".. the goal of this simulation" part. I think "To achieve this" might be better actually, but either way, a transitional statement is needed, rather than directly starting with the steps of the simulation. 

I'm confused by your second comment. I don't have ", and" in my suggestions. I don't think the original three bullets convey the idea we're doing correctly (and I'm talking about the core idea, not the semantics of how we communicate it), so I'm not sure there is a point in reviewing that. 

I see your preference for instruction style, but disagree that it's the way to go here. Try re-writing what I wrote with instruction style -- it reads weird to me. Besides, the goal is not to give instructions here, the goal is to describe what we did to illustrate our point, so I don't necessarily see what it should be instruction-style.

Reply:
@hassan-obeid ah I see our confusion now.

In @bouzaghrane's [file](https://github.com/hassan-obeid/tr_b_causal_2020/blob/abdb28bb0e8d5ba21f2da74986d1cab5b4bc0c5f/article/sections/_2_graph_importance.tex), lines 93 - 94 say:
> For our purposes, we assume that these DAGs represent the ``true'' data generating process, where several variables affect each other.
Each of these graphs is based on the utility function of each mode in the multinomial logit model specified in \citet{brathwaite_asymmetric}.

Your [comment](https://github.com/hassan-obeid/tr_b_causal_2020/pull/211#discussion_r550759730) for line 93 says ".. the goal of this simulation".
We were looking at two different line 93s.
Besides this, we're in total agreement about the need for transitional statements.

", and" is in the original bullet points.
Sorry for the confusion caused by including the last paragraph in my earlier comment.
I should have put it in its own comment instead of placing it above.

Reply:
I am somewhere in the middle of both of you on this. I say that in line 124 we replace it by "To achieve this goal, we follow these simulation steps, using the same outcome model:" and keep the original "Simulate, modify, predict" structure in the bullets with the second bullet broken down like you did.

@hassan-obeid I also suggest dropping the "representing the true data generating process" from the first bullet since it was already mentioned in your suggestion in line 91.

Reply:
```suggestion
To achieve this goal, we follow these simulation steps, using the same outcome model:
\begin{itemize}
   \item Simulate data from the DAGs shown in Figure \ref{fig:IND_GRAPH} through Figure \ref{fig:SR3_causal_2}
   \item In one scenario, modify the travel distance variable in all utility graphs to emulate a company's decision to move its employees closer to campus. This is similar to assuming the DAG in Figure \ref{fig:IND_GRAPH} as the true data generating model.
   \item In the other scenario, modify the travel distance variable in all utility graphs, as well as all the variables that are dependent on it based on the DAGs in Figure \ref{fig:IND_GRAPH} through Figure \ref{fig:SR3_causal_2}. This case is meant to reflect the "correct" approach needed to quantify the effects of the companies policy. 
   \item Predict the probabilities of choosing car-centric modes before and after modifying the data to emulate the proposed intervention under each of the two scenarios above, and compute the differences in mode choice probabilities.
\end{item}
```

Reply:
@bouzaghrane I think that the way you rephrased bullet 2 de-emphasizes the main point -- it's not wrong, just the emphasis is not where we want it. My preference is to keep it the way I wrote it, as it emphasizes the fact that we only modified travel distance. The way you wrote it, it's left for the reader to figure out that we only modified travel distance, and we're assuming that they'll look at the IND_GRAPH figure and realize that. 
Besides, I find it a bit off to say "Modify the travel distance variable based on a DAG". _Modifying_ a variable is independent of a DAG. That's why I wrote "modify travel distance **_only_**", and "this assumes the IND_GRAPH as the true DAG". The **_only_** part is where the DAG gets involved. 

cc @timothyb0912 for visibility.

Reply:
@hassan-obeid Hmm I see. I implemented the change in here. I also think that the last sentence in the third bullet is somewhat incorrect. Bullet 3 shows the correct approach for finding out what the data would look like under the true data generating process, not necessarily what the effect of the proposed policy is. I think we're all on the same page and it's just somewhat of a semantics issue here.

@timothyb0912 I'm done with all changes except this one. I will commit this version but leave the comment open for visibility and reference in your final review.

```suggestion
To achieve this goal, we follow these simulation steps, using the same outcome model:
\begin{itemize}
   \item Simulate data from the DAGs shown in Figure \ref{fig:IND_GRAPH} through Figure \ref{fig:SR3_causal_2}.
   \item In one scenario, only modify the travel distance variable in all graphs to emulate a company's decision to move its employees closer to campus. This is similar to assuming the DAG in Figure \ref{fig:IND_GRAPH} as the true data generating model.
   \item In the other scenario, modify the travel distance variable in all utility graphs, as well as all the variables that are dependent on it based on the DAGs in Figure \ref{fig:DA_causal_2} through Figure \ref{fig:SR3_causal_2}. This case is meant to reflect the ``correct'' approach needed to quantify the effects of the company's policy. 
   \item Predict the probabilities of choosing car-centric modes before and after modifying the data to emulate the proposed intervention under each of the two scenarios above, and compute the differences in mode choice probabilities.
\end{itemize}
```

Reply:
@hassan-obeid let me know if this change I suggested is fair game.